src/components/PostDetails.tsx

```diff
@@ -1,117 +1,110 @@
-import React from 'react';
+import { useContext, useEffect, useState } from 'react';
 import { Loader } from './Loader';
+import { PostsContext } from '../context/PostsContext';
+import { deleteCommentById, getCommentsById } from '../api/comments';
 import { NewCommentForm } from './NewCommentForm';
+import { CommentsContext } from '../context/CommentsContext';
 
 export const PostDetails: React.FC = () => {
+  const { post } = useContext(PostsContext);
+  const { comments, setComments } = useContext(CommentsContext);
```

Comment:
You can create a hook for your contexts and in this case, you don't need to pass context every time:
```javascript
const useComments = () => useContext(CommentsContext);
```

Reply:
I don't really understand how to do this, so I'll leave it like that for now